creating_diverse_inclusive_communities.adoc

```diff
@@ -115,124 +124,147 @@ It shows a lack of transparency that wastes time, sows distrust, and prevents ma
+The first step to a more inclusive open source project involves bursting the meritocracy bubble: the more you believe in meritocracy, the more biased your project is https://www2.deloitte.com/au/en/blog/diversity-inclusion-blog/2019/meritocracy-unraveling-paradox.html[likely to be].
 
-Why? Purely meritocratic projects https://gap.hks.harvard.edu/paradox-meritocracy-organizations[don't acknowledge] that people enter on unequal playing fields.
-If an open source maintainer isn't aware that women often  have less time to contribute, or that LGBTQ+ contributors are more likely to endure online abuse, they won't take steps to make the community more inclusive.
+Why?
+Purely meritocratic projects https://gap.hks.harvard.edu/paradox-meritocracy-organizations[don't acknowledge] that people enter on unequal playing fields.
+If an open source maintainer isn't aware that women often have less time to contribute, or that LGBTQ+ contributors are more likely to endure online abuse, they won't take steps to make the community more inclusive.
 As a result, they risk losing the diverse contributors they worked hard to recruit.
+
 Terri Oda (She/Her) volunteers for the Python Software Foundation and Google's Summer of Code alongside her role as an open source security researcher at Intel.
-She says claims of meritocracy make her cringe.
-Why?
-Such statements cause maintainers to ignore opportunities to get more people involved in projects, even in cases where the open source community gathers in person.
+She says claims of meritocracy make her cringe, she says, because they cause maintainers to ignore opportunities to get more people involved in projects, even in cases where the open source community gathers in person.
+
 "For example, say you're running code sprints at a conference and want to increase the number of women," Oda says.
 "If you're thinking about merit and skills, you're going to wind up offering more intro-to-sprinting type content or whining about the university pipeline.
 But if you look at the bigger picture, you might realize that the conference offers childcare during the main conference, but it stops when sprints start.
 Or that the venue isn't in a safe area and the sprints run until after dark."
+
 The first step to build a more inclusive environment is self-awareness.
 Open source contributors enter projects with a range of lived experiences that affect how—and if—they show up.
 Sitting with and reflecting on this fact is the first, most crucial step.
+
 The next step is to take an honest look at your project's current community, and take note of who is—and isn't—there.
 If your project contributors all, or even mostly, look like you, that's a huge red flag that an inclusive overhaul is in order. 
 
 === 2. Prioritize your project's documentation
 
 A 2019 Stack Overflow study found that about 41% of developers have less than five years of experience.
```

Comment:
Do you happen to have a link to this study?

Reply:
Here it is! https://insights.stackoverflow.com/survey/2019#:~:text=Over%2040%25%20of%20respondents%20have,and%20Kotlin%20have%20the%20fewest.

Reply:
Here it is! https://insights.stackoverflow.com/survey/2019#:~:text=Over%2040%25%20of%20respondents%20have,and%20Kotlin%20have%20the%20fewest.

Reply:
PR is updated, thanks!